src/init.cpp

```diff
@@ -1718,9 +1718,20 @@ bool AppInit2(boost::thread_group& threadGroup, CScheduler& scheduler)
 
         //Set Transaction Deletion Options
         fTxDeleteEnabled = GetBoolArg("-deletetx", false);
+        fTxConflictDeleteEnabled = GetBoolArg("-deleteconlicttx", true);
```

Comment:
typo `-deleteconlicttx` should be `-deleteconflicttx`

Reply:
fixed